### ISSUE TITLE
pkg_resources.DistributionNotFound: pyzmq==14.7.0

### ISSUE THREAD

**Issue by f0b04707** [tingbot/tbprocessd#2]
tbprocessd does not start with version of pyzmq > 14.7.0. The error is the following:
`pkg_resources.DistributionNotFound: pyzmq==14.7.0`

Installed version of pyzmq:
`Successfully installed docopt-0.6.2 pyzmq-15.2.0 virtualenv-15.0.2`

`setup.py` of ​**tbprocessd** needs to be updated for `install_requires` with **>=** instead of **==**

```
from setuptools import setup
setup(
    name="tbprocessd",
    py_modules=['tbprocessd', 'tbopen', 'tbtail'],
    version='0.2',
    entry_points={'console_scripts': [
        'tbprocessd = tbprocessd:main',
        'tbopen = tbopen:main',
        'tbtail = tbtail:main',
    ]},
    install_requires=[
        'pyzmq>=14.7.0',
        'requests'
    ],
)
```

**Comment by f0b04707** [comment]
Fixed in #5

**Comment by joerick** [comment]
👌
